udata_front/theme/gouvfr/datagouv-components/src/components/DatasetCard/DatasetCard.vue

```diff
@@ -33,74 +33,76 @@
         </div>
       </div>
       <div class="fr-col">
-        <h4 class="fr-mb-1v fr-grid-row">
+        <h4 class="fr-text--md fr-mb-0 fr-grid-row">
           <slot name="datasetUrl" :dataset="dataset" :datasetUrl="datasetUrl">
             <AppLink :to="datasetUrl" class="text-grey-500">
               {{ dataset.title }}
               <small v-if="dataset.acronym">{{ dataset.acronym }}</small>
             </AppLink>
           </slot>
         </h4>
-        <p class="fr-m-0 fr-text--sm" v-if="dataset.organization || dataset.owner">
-          {{ t('From') }}
+        <p class="fr-my-0 fr-text--sm" v-if="dataset.organization || dataset.owner">
           <template v-if="dataset.organization">
-            <span class="not-enlarged" v-if="organizationUrl">
-              <AppLink :to="organizationUrl" class="fr-link">
+            <span class="not-enlarged dash-after fr-mr-1v" v-if="organizationUrl">
+              <AppLink class="fr-link fr-text--sm" :to="organizationUrl">
                 <OrganizationNameWithCertificate :organization="dataset.organization" />
               </AppLink>
             </span>
             <OrganizationNameWithCertificate v-else :organization="dataset.organization" />
           </template>
-          <template v-else>{{ownerName}}</template>
+          <span class="not-enlarged dash-after fr-mr-1v" v-else>
```

Comment:
There are discrepancies between dashes in Figma and the current implementation (too big, black instead of grey)